tests/test_generate_authentication_options.py

```diff
@@ -25,7 +26,7 @@ def test_generates_options_with_custom_values(self) -> None:
         options = generate_authentication_options(
             rp_id="example.com",
             allow_credentials=[
-                PublicKeyCredentialDescriptor(id=b"12345"),
+                PublicKeyCredentialDescriptor(id=urlsafe_b64encode(b"12345")),
```

Comment:
`b"12345"` and `urlsafe_b64encode(b"12345")` aren't equivalent, the `id` value needs to be `bytes` because it's the `options_to_json` that encodes `bytes` to base64url-encoded `str`ings...maybe I should come back to tests later once I see the changes to the actual modules...